packages/node-id/src/createNodeIdPlugin.ts

```diff
@@ -100,9 +99,13 @@ export const withNodeId = ({
       (!filterText || (operation.properties as Partial<TNode>).type)
```

Comment:
```suggestion
      idKey in operation.properties
```
I guess this case runs only when there is an id, so no need to filter

Reply:
do we need this parameter then? with this check removed, there's no way an id will be added to leafs nodes.

Reply:
`filterText` is used above